provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/task/RequestData.java

```diff
@@ -7,20 +7,43 @@
 
 package com.microsoft.azure.sdk.iot.provisioning.device.internal.task;
 
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+
 import javax.net.ssl.SSLContext;
 
 @SuppressWarnings("SameParameterValue") // Some constructors currently are passed null params, but they are designed to be generic.
 public class RequestData
 {
+    @Getter
+    @Setter(AccessLevel.MODULE)
```

Comment:
same question that I had in the other PR, module vs package access level

Reply:
I'll mark these all as Package, sure